notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -0,0 +1,1364 @@
+{
```

Comment:
<ul><li>"simply convenience class" -&gt; "simple convenience class"</li><li>I suggest removing that sentence above, as long as the cell below, which seem not necessary. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>I think this <code>NoBaseClassMovielensModel</code> can support the understanding of tfrs.model based model which is highly abstracted and looks like a blackbox.</p><p>What do you think?</p><p> </p><p>Another option could be move the <code>NoBaseClassMovielensModel</code> from an executable cell to a markdown to show it is not needed to be run. </p><p> </p>